### ISSUE TITLE
Explain how to get started with federation

### ISSUE THREAD

**Issue by Nutomic** [LemmyNet/lemmyBB#36]
To start federating with another instance, you need to paste its url (or `!name@example.com`) into the search field. The same works for users, posts and comments. This works identically across the Fediverse, but is very obscure for new users. 

Do you have any idea how this functionality could be explained to new users? Are there other Fediverse projects which handle it well?

**Comment by Tealk** [comment]
I have tried it in my manual like this
https://handbuch.rollenspiel.monster/mastodon/ui/simple.html?highlight=suche#suche

**Comment by Nutomic** [comment]
Yes we could integrate a [FAQ](https://www.phpbb.com/community/help/faq).
